index.d.ts

```diff
@@ -0,0 +1,13 @@
+import {Primitive, JsonObject} from 'type-fest';
+
+export type ErrorObject = {
+	name?: string;
+	stack?: string;
+	message?: string;
+	code?: string;
+} & JsonObject;
+
+/**
+Serialize an error into a plain object.
+*/
```

Comment:
Needs an example like this: https://github.com/sindresorhus/serialize-error/commit/a3587ee620b54be4935492b572a17f5cf42fe2b2